.github/workflows/sync-issue-to-jira.yml

```diff
@@ -0,0 +1,25 @@
+name: Sync a given GitHub Issue to Jira
+
+on:
+  issues:
+    types: [labeled]
+
+jobs:
+  sync-issue:
+    if: secrets.JIRA_BASE_URL != '' && secrets.JIRA_USER_EMAIL != '' && secrets.JIRA_API_TOKEN != '' && github.event.label.name == 'sync-to-jira'
+    name: Sync GitHub Issue to Jira
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
+          cache: "pip" # caching pip dependencies
+
+      - name: Install python requirements
+        run: |
+          pip install --upgrade pip && pip install -r scripts/requirements.txt
+
+      - name: Run link script
+        run: |
+          python3 jira_driver.py --issue ${{github.event.issue.number}} link
```

Comment:
Need to explicitly set the environment variables to their corresponding values in secrets.